package.json

```diff
@@ -15,20 +15,20 @@
   "version": "0.1.2",
   "dependencies": {
     "github": "0.2.4",
-    "d3-queue": "2.0.3",
-    "lambda-cfn": "^1.0.0"
+    "d3-queue": "^3.0.0",
+    "@mapbox/lambda-cfn": "^2.0.0"
   },
   "scripts": {
-    "test": "NODE_ENV=test tape test/*.test.js test/**/*.test.js",
-    "pretest": "eslint index.js"
+    "test": "eslint **/*.js && NODE_ENV=test tape test/*.test.js test/**/*.test.js",
+    "lint": "eslint **/*.js",
+    "unit-test": "NODE_ENV=test tape test/*.test.js test/**/*.test.js"
   },
   "devDependencies": {
-    "nock": "^7.5.0",
-    "tape": "^4.5.1",
-    "eslint": "^2.4.0",
+    "nock": "^9.0.0",
+    "tape": "^4.0.0",
+    "eslint": "^4.0.0",
     "eslint-config-unstyled": "^1.1.0"
```

Comment:
@zmully - are we using eslint-config-unstyled in this project? I don't believe we are, might as well remove it in this PR too since while we're upgrading dependencies.